www/templates/default/html/js/events.js

```diff
@@ -4,6 +4,15 @@ require(['jquery', 'wdn', 'modernizr'], function($, WDN, Modernizr) {
 	var $progress = $('<progress>'),
 		mqBp2 = '(min-width: 768px)';
 	
+	$(document).ready(function() {
```

Comment:
We should probably have a separate file for manager js.  There is no need to load manager only javascript in the front end.